Runtime/Client/LootLockerEndPoints.cs

```diff
@@ -233,6 +234,25 @@ public class LootLockerEndPoints
         public static EndPointClass ComputeAndLockDropTable = new EndPointClass("v1/player/droptables/{0}/compute?asset_details={1}", LootLockerHTTPMethod.POST);
         public static EndPointClass PickDropsFromDropTable = new EndPointClass("v1/player/droptables/{0}/pick", LootLockerHTTPMethod.POST);
 
+        // Currencies
+        [Header("Currencies")]
+        public static EndPointClass listCurrencies = new EndPointClass("currency", LootLockerHTTPMethod.GET);
+        public static EndPointClass getCurrencyByCode = new EndPointClass("currency/code/{0]", LootLockerHTTPMethod.GET);
+        public static EndPointClass getCurrencyDenominations = new EndPointClass("currency/{0}/denominations", LootLockerHTTPMethod.GET);
+
+        // Balances
+        [Header("Balances")]
+        public static EndPointClass listBalancesInWallet = new EndPointClass("balances/wallet/{0}", LootLockerHTTPMethod.GET);
+        public static EndPointClass getWalletByWalletId = new EndPointClass("wallets/{0]", LootLockerHTTPMethod.GET);
```

Comment:
Same thing here. Unsure if this is on purpose

Reply:
Fixed